pkg/testutil/rand/model.go

```diff
@@ -67,13 +67,19 @@ func WithID(id uuid.UUID) ModelServiceOption {
 
 // modelMessage generates a random model.Message.
 func modelMessage() *model.Message {
-	return &model.Message{
+	msg := &model.Message{
 		ID:          gofakeit.SentenceSimple(),
 		PluralID:    gofakeit.SentenceSimple(),
 		Message:     gofakeit.SentenceSimple(),
 		Description: gofakeit.SentenceSimple(),
 		Fuzzy:       gofakeit.Bool(),
 	}
+
+	if gofakeit.Bool() {
+		gofakeit.Slice(&msg.Positions)
+	}
```

Comment:
```suggestion
	gofakeit.Slice(&msg.Positions)
```

Reply:
that's there on purpose, create nil value